### ISSUE TITLE
ModuleNotFoundError: No module named 'urlparse'

### ISSUE THREAD

**Issue by suresh-nataraj** [heroku/kafka-helper#6]
I am trying to use python-helper and it is failing with below error:

Traceback (most recent call last):
  File "kafka_consumer.py", line 1, in <module>
    import kafka_helper
  File "/usr/local/lib/python3.6/site-packages/kafka_helper.py", line 9, in <module>
    from urlparse import urlparse
ModuleNotFoundError: No module named 'urlparse'

**Comment by apotapov** [comment]
`urlparse` was moved to `urllib.parse` in Python3

Something along the lines of this would do the trick:
```
try:
    import urlparse
except ImportError:
    import urllib.parse as urlparse
```
 I'll try putting a PR together, although I suspect there might be other Python3 issues

**Comment by apotapov** [comment]
Looks like this was fixed on master but not deployed?

**Comment by apotapov** [comment]
Installing with `pip install -e git+https://github.com/heroku/kafka-helper@70aa539540e7d6517f0c3c1900617767ab198b6c#egg=kafka_helper` does the trick.

**Comment by alouie-sfdc** [comment]
Thanks -- this should be fixed with the release of kafka-helper 0.2: https://pypi.org/project/kafka-helper/0.2/

**Comment by madux** [comment]
Using python3 .. 
from urllib.parse import parse
